config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  root 'homepages#index'
+
+  resources :drivers
+  resources :trips
+
+  resources :passengers do
+    resources :trips, only: [:index, :new, :create]
```

Comment:
`trips#new` shouldn't live here since the passenger isn't treated specially when you make a new trip.